swift-sdk/IterableConfig.swift

```diff
@@ -55,6 +55,13 @@ import Foundation
 /// The delegate for getting the authentication token
 @objc public protocol IterableAuthDelegate: AnyObject {
     @objc func onAuthTokenRequested(completion: @escaping AuthTokenRetrievalHandler)
+    @objc func onTokenRegistrationFailed()
```

Comment:
Can failure be called with a string so that a reason can be sent?

Reply:
Yep!